tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -380,29 +260,6 @@ class WskBasicUsageTests extends TestHelpers with WskTestHelpers {
       }
   }
 
-  it should "report error when creating an action with unknown kind" in withAssetCleaner(wskprops) {
-    (wp, assetHelper) =>
-      val rr = assetHelper.withCleaner(wsk.action, "invalid kind", confirmDelete = false) { (action, name) =>
-        action.create(
-          name,
-          Some(TestUtils.getTestActionFilename("echo.js")),
-          kind = Some("foobar"),
-          expectedExitCode = BAD_REQUEST)
-      }
-      rr.stderr should include regex "kind 'foobar' not in Set"
-  }
-
-  it should "report error when creating an action with zip but without kind" in withAssetCleaner(wskprops) {
```

Comment:
I think this test is applicable with wskRest

Reply:
done.